modules/renter/downloadcache.go

```diff
@@ -0,0 +1,53 @@
+package renter
```

Comment:
Yeah that would be pretty cool. Especially in combination with the PR mentioned above. That would allow users to stream multiple videos in parallel efficiently if they have enough ram available.